ej2-react/datepicker/getting-started.md

```diff
@@ -1,14 +1,14 @@
 ---
 layout: post
-title: Getting started with React Datepicker component | Syncfusion
-description:  Checkout and learn about Getting started with React Datepicker component of Syncfusion Essential JS 2 and more details.
+title: Getting started with React DatePicker Component | Syncfusion
+description:  Checkout and learn about Getting started with Syncfusion Essential React DatePicker component, its elements and more details.
```

Comment:
Revert these description changes and remove extra spaces.

Reply:
Reverted